src/popup/components/TransferSendForm.vue

```diff
@@ -2,6 +2,11 @@
   <div class="transfer-send-form">
     <template v-if="isMultisig">
       <ModalHeader :title="$t('modals.multisigTxProposal.title')" />
+      <InfoBox
+        v-if="hasMultisigTokenWarning"
+        :type="INFO_BOX_TYPES.warning"
+        :text="$t('modals.multisigTxProposal.tokenWarning')"
+      />
```

Comment:
Why not just disable the possibility of changing the asset if it's a multisig?

Reply:
It is disabled, but if someone scans different token they can be confused why it was changed to AE

Reply:
OK, makes sense.